src/utils.ts

```diff
@@ -0,0 +1,18 @@
+const TOKEN_KEY = 'jwt';
+
+export const login = () => {
+    localStorage.setItem(TOKEN_KEY, 'TestLogin');
+};
+
+export const logout = () => {
+    localStorage.removeItem(TOKEN_KEY);
+};
+
+export const isLogin = () => {
+    return true;
+    // if (localStorage.getItem(TOKEN_KEY)) {
```

Comment:
Uncomment and refactor. `() => !!localStorage.getItem(TOKEN_KEY)`

Reply:
Done :)